azure-devops/azext_devops/dev/team/team.py

```diff
@@ -0,0 +1,158 @@
+    :type devops_organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: When 'On' unsupplied arg values will be detected from the current working
+                   directory's repo.
+    """
+    try:
+        devops_organization, project = resolve_instance_and_project(detect=detect,
+                                                                    devops_organization=devops_organization,
+                                                                    project=project)
+        core_client = get_core_client(devops_organization)
+        return core_client.delete_team(team_id=team_id, project_id=project)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+
+def get_team(team_id, devops_organization=None, project=None, detect=None):
+    """Gets a team.
+    :param team_id: The name or id of the team to show.
+    :type team_id: str
+    :param devops_organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type devops_organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: When 'On' unsupplied arg values will be detected from the current working
+                   directory's repo.
+    :rtype: :class:`<WebApiTeam> <core.v4_0.models.WebApiTeam>`
+    """
+    try:
+        devops_organization, project = resolve_instance_and_project(detect=detect,
+                                                                    devops_organization=devops_organization,
+                                                                    project=project)
+        core_client = get_core_client(devops_organization)
+        return core_client.get_team(team_id=team_id, project_id=project)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+
+def get_teams(top=None, skip=None, devops_organization=None, project=None, detect=None):
+    """List of all teams for a project.
+    :param top: Maximum number of teams to return.
+    :type top: int
+    :param skip: Number of teams to skip.
+    :type skip: int
+    :param devops_organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type devops_organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: When 'On' unsupplied arg values will be detected from the current working
+                   directory's repo.
+    :rtype: [WebApiTeam]
+    """
+    try:
+        devops_organization, project = resolve_instance_and_project(detect=detect,
+                                                                    devops_organization=devops_organization,
+                                                                    project=project)
+        core_client = get_core_client(devops_organization)
+        return core_client.get_teams(top=top, skip=skip, project_id=project)
```

Comment:
We had that long discussion on validating inputs like top, skip > 0. Are we planning to follow it?

Reply:
To be honest, I would prefer to have very basic input validations which are not api specific and having below validations in CLI looks convincing to me.
(1) trimming a string and validating if its length is greater than ZERO
(2) value greater than zero for parameters like top or skip

Reply:
Yeah, makes sense to have basic validations across most commands. Opened #331 for the same.